system/security/shared/seclib.hpp

```diff
@@ -78,6 +78,23 @@ inline const char * getSecAccessFlagName(SecAccessFlags flag)
     }
 }
 
+inline SecAccessFlags getSecAccessFlag(const char* name)
+{
+    if (isEmptyString(name))
+        return SecAccess_Unavailable;
```

Comment:
Term unavailable seems slightly confusing, but you could convince me.

Reply:
translateAuthLevel, in esdl_def.cpp, maps an empty string to SecAccess_Full, and EsdlServiceImpl::configureBindingAccessMap maps SecAccess_Unavailable to SecAccess_Full. My security manager plugin, however, considers SecAccess_Unavailable to be an error.

Maybe the plugin is wrong and should also treat SecAccess_Unavailable like SecAccess_Full (hint: this is a request for an opinion). But I'm not convinced that a generic utility should assume that every caller wants this equivalence. The generic behavior should indicate that nothing was given, allowing the caller to treat the condition in whatever way it chooses.

Reply:
Let's discuss this afternoon.  At the level of this function my main concern was that if you squint your eyes hard enough, the term "Unavailable" starts to sound like it has a real security meaning.  Rather than just meaning that the string was empty.  But I'd like to better understand each context that this might be called now and in the future.